docker/build_jar.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
add a file-level comment for what this does and where it might be called. I'm assuming it's called in terra-github-workflows.

Reply:
Updated comment.